src/web/components/ShareIcons.tsx

```diff
@@ -59,102 +59,131 @@ const shareIcon = (colour: string) => css`
 	}
 
 	:hover {
-		background-color: ${colour};
-		border-color: ${colour};
+		background-color: ${pillarPalette[pillar].main};
+		border-color: ${pillarPalette[pillar].main};
 		fill: white;
 	}
 `;
 
-const mobileOnlyShareIconsListItem = css`
-	${from.phablet} {
-		display: none;
-	}
-`;
-
-interface ShareListItemType {
-	id: SharePlatform;
-	Icon: React.ComponentType;
-	url: string;
-	userMessage: string;
-	mobileOnly: boolean;
-}
-
-export const SharingIcons: React.FC<{
-	sharingUrls: {
-		[K in SharePlatform]?: {
-			url: string;
-			userMessage: string;
-		};
-	};
+export const ShareIcons: React.FC<{
+	pageId: string;
+	webTitle: string;
 	displayIcons: SharePlatform[];
 	pillar: Theme;
-	className?: string;
-}> = ({ sharingUrls, displayIcons, pillar, className }) => {
-	const icons: { [K in SharePlatform]?: React.ComponentType } = {
-		facebook: FacebookIcon,
-		twitter: TwitterIconPadded,
-		email: EmailIcon,
-		linkedIn: LinkedInIcon,
-		pinterest: PinterestIcon,
-		whatsApp: WhatsAppIcon,
-		messenger: MessengerIcon,
-	};
-
-	const mobileOnlyIcons: SharePlatform[] = ['whatsApp', 'messenger'];
-
-	const shareList = displayIcons.reduce((list: ShareListItemType[], id) => {
-		const icon = icons[id];
-		const sharingUrl = sharingUrls[id];
-
-		if (icon && sharingUrl) {
-			const listItem: ShareListItemType = {
-				id,
-				Icon: icon,
-				mobileOnly: mobileOnlyIcons.includes(id),
-				...sharingUrl,
-			};
-			list.push(listItem);
-		}
-
-		return list;
-	}, []);
-
+}> = ({ pageId, webTitle, displayIcons, pillar }) => {
 	return (
-		<ul className={cx(shareIconList, [className])}>
-			{shareList.map((shareListItem) => {
-				const {
-					Icon,
-					id,
-					url,
-					userMessage,
-					mobileOnly,
-				} = shareListItem;
-
-				return (
-					<li
-						className={cx(shareIconsListItem, {
-							[mobileOnlyShareIconsListItem]: mobileOnly,
-						})}
-						key={`${id}Share`}
+		<ul className={ulStyles}>
+			{displayIcons.includes('facebook') && (
+				<li className={liStyles} key="facebook">
+					<a
+						href={`https://www.facebook.com/dialog/share?app_id=202314643182694&href=https://www.theguardian.com/${pageId}&CMP=share_btn_fb`}
+						role="button"
+						aria-label="Share on Facebook"
+						target="_blank"
+					>
+						<span className={iconStyles(pillar)}>
+							<FacebookIcon />
+						</span>
+					</a>
+				</li>
+			)}
+
+			{displayIcons.includes('twitter') && (
+				<li className={liStyles} key="twitter">
+					<a
+						href={`https://twitter.com/intent/tweet?text=${webTitle.replace(
+							/Leave.EU/gi,
+							'Leave. EU',
+						)}&url=https://www.theguardian.com/&CMP=share_btn_tw`}
```

Comment:
I think the URL here is broken. It's always linking back to the homepage

Reply:
👏 for spotting this, that turned out to be an issue on the site. Wish I hadn't merged this in so quickly!